libraries/Microsoft.Bot.Builder.Runtime/ConfigurationConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Runtime
+{
+    public static class ConfigurationConstants
```

Comment:
All public members need XML comments.

Reply:
This is temporary. We have a P0 issue to resolve missing comments in R12.